ReactCommon/cxxreact/Instance.h

```diff
@@ -62,9 +62,10 @@ class RN_EXPORT Instance {
 
   void setSourceURL(std::string sourceURL);
```

Comment:
>ng [](start = 29, length = 2)

this should be && too

Reply:
Not wanting to change existing interfaces for upstream RN. See other comment.

---
In reply to: [342831087](https://github.com/microsoft/react-native/pull/187#discussion_r342831087) [](ancestors = 342831087)